packages/adapters-library/src/core/utils/getTokenMetadata.ts

```diff
@@ -52,8 +52,8 @@ export async function getTokenMetadata(
     return onChainTokenMetadata
   }
 
-  const errorMessage = 'Cannot find token metadata for token'
-  logger.error({ tokenAddress, chainId }, errorMessage)
+  const errorMessage = 'Failed getting token metadata'
+  logger.error({ tokenAddress, chainId })
```

Comment:
It'll be difficult to know what error it is in the logs with just the object, it should also include the message.